modules/effects/src/main/scala/dev/profunktor/redis4cats/effects.scala

```diff
@@ -298,13 +298,29 @@ object effects {
   }
 
   object XReadOffsets {
+
+    def all[K](keys: K*): Set[XReadOffsets[K]]                    = All(keys: _*).map(identity)
+    def latest[K](keys: K*): Set[XReadOffsets[K]]                 = Latest(keys: _*).map(identity)
+    def custom[K](offset: String, keys: K*): Set[XReadOffsets[K]] = Custom(offset, keys: _*).map(identity)
+
     case class All[K](key: K) extends XReadOffsets[K] {
       override def offset: String = "0"
     }
+    object All {
+      def apply[K](keys: K*): Set[All[K]] = keys.map(k => new All[K](k)).toSet
```

Comment:
These apply methods are a bit unconventional as they are not creating XReadOffsets[K] rather Sets

Reply:
Yeah, I was struggling w/ names, any suggestions? I was thinking `of` or `for`.